library/OrderCloud.Catalyst/Integrations/Implementations/Vertex/Mappers/VertexRequestMapper.cs

```diff
@@ -0,0 +1,104 @@
+﻿using OrderCloud.SDK;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace OrderCloud.Catalyst
+{
+	public static class VertexRequestMapper
+	{
+		public static VertexCalculateTaxRequest ToVertexCalculateTaxRequest(this OrderWorksheet order, List<OrderPromotion> promosOnOrder, string companyCode, VertexSaleMessageType type)
```

Comment:
I included that because a different Tax provider, Avalara, has specific fields for tracking discounts on an order, so that data is needed. Even though Vertex can't use that data, I'm trying to standardize a tax interface.

Reply:
Resolved. Now using LineItem and Order PromotionDiscount fields instead.